tests/openstack/task.yaml

```diff
@@ -0,0 +1,101 @@
+summary: Test the openstack backend.
+
+systems: [ubuntu-22.04-64-devstack]
+
+kill-timeout: 30m
+
+prepare: |
+    if [ ! -f .spread-reuse.yaml ]; then
+        touch /run/spread-reuse.yaml
+        ln -s /run/spread-reuse.yaml .spread-reuse.yaml
+    fi
+
+    sudo -u stack -i /opt/stack/devstack/unstack.sh
+    sudo -u stack -i /opt/stack/devstack/stack.sh
```

Comment:
Where are these coming from?

Reply:
devstack is preinstalled in the image, so when we start the tests we just prepare it.

The alternative is to download/configure and start devstack in prepare section, but it makes the test slower.